src/actions/src.cr

```diff
@@ -1,13 +1,29 @@
 module DppmRestApi::Actions::Src
   extend self
   include RouteHelpers
-  relative_get "" do |context|
+
+  # Lists available sources.
```

Comment:
This renders #27 redundant :thinking:

Reply:
Only very partially. I have only done one route for listing available sources here.
We will need source package information too, and listing them.